packages/appstore/src/modules/traders-hub/account-without-wallets.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
@sergei-deriv These file will be removed from code after releasing the wallets project.
let's leave a comment here for this